libs/blocks/commerce/commerce.js

```diff
@@ -125,49 +89,43 @@ export const decorateOfferDetails = async (el, of, searchParams) => {
   offerDetailsList.appendChild(clearButton);
   offerDetailsList.appendChild(checkoutUrl);
   el.append(offerDetailsList);
-};
+}
 
-export const handleSearch = async (event, el) => {
-  const displaySearchError = () => {
-    const notValidUrl = createTag('h4', { class: 'not-valid-url' }, 'Not a valid offer link');
-    el.append(notValidUrl);
-  };
+export async function handleSearch(event, el) {
```

Comment:
Maybe rename it to a more specific handleOfferSearch

Reply:
Renamed, thanks.